frontend/test/__runner__/cypress-runner-backend.js

```diff
@@ -31,6 +31,7 @@ const CypressBackend = {
       ];
 
       const metabaseConfig = {
+        MB_RUN_MODE: "test",
```

Comment:
This is so that we can accurately detect when we're in a Cypress test using `config/is-test?`. Not sure why this wasn't set before—probably was just never required.